Applications/CMC/test/testCMCArm26_Millard.cpp

```diff
@@ -45,15 +45,26 @@ void testCMCArm26() {
     Storage *standard = new Storage();
     cmc.getModel().formStateStorage(temp, *standard);
 
-    Array<double> rms_tols(0.02, 2*2+2*6); // activations within 2%, angles within .6 degrees
+    Array<double> rms_tols(0, 2*2+2*6); 
     const string& muscleType = cmc.getModel().getMuscles()[0].getConcreteClassName();
     string base = "testCMCArm26 "+ muscleType;
 
-    rms_tols[6] = 0.05;
-    rms_tols[8] = 0.05;
-    rms_tols[10] = 0.05;
-    rms_tols[12] = 0.05;
-    rms_tols[14] = 0.05;
+    rms_tols[0]  = 0.0005; // r_shoulder/r_shoulder_elev/value
+    rms_tols[1]  = 0.003;  // r_shoulder/r_shoulder_elev/speed
+    rms_tols[2]  = 0.002;  // r_elbow/r_elbow_flex/value
+    rms_tols[3]  = 0.008;  // r_elbow/r_elbow_flex/speed
+    rms_tols[4]  = 0.002;  // TRIlong/activation
+    rms_tols[5]  = 0.0006; // TRIlong/fiber_length
+    rms_tols[6]  = 0.03;   // TRIlat/activation
+    rms_tols[7]  = 0.0003; // TRIlat/fiber_length
+    rms_tols[8]  = 0.03;   // TRImed/activation
+    rms_tols[9]  = 0.0003; // TRImed/fiber_length
+    rms_tols[10] = 0.05;   // BIClong/activation
+    rms_tols[11] = 0.0008; // BIClong/fiber_length
+    rms_tols[12] = 0.0071; // BICshort/activation
+    rms_tols[13] = 0.0005; // BICshort/fiber_length
+    rms_tols[14] = 0.003;  // BRA/activation
+    rms_tols[15] = 0.0001; // BRA/fiber_length
 
     CHECK_STORAGE_AGAINST_STANDARD(results, *standard, rms_tols, __FILE__, __LINE__, 
```

Comment:
It is not necessary to individualize the tolerances.  The original test gave the numerical rationale for the values chosen. Being within 1 degree of the standard angles and with 5% of the Millard muscles seems fine to me. The standard was created using the Thelen muscle so we would not expect the answers to be identical.